pkg/minikube/constants/constants_linux.go

```diff
@@ -18,7 +18,14 @@ limitations under the License.
 
 package constants
 
+import (
+	"k8s.io/client-go/util/homedir"
+)
+
 var SupportedVMDrivers = [...]string{
 	"virtualbox",
 	"kvm",
 }
+
+var DefaultMountDir = homedir.HomeDir()
```

Comment:
Maybe we can just switch this to `constants.DefaultMinipath`?

You probably only want to delete your mounts for the minikube profile you're working on.